product_docs/docs/biganimal/release/getting_started/creating_a_cluster/index.mdx

```diff
@@ -213,16 +214,82 @@ Enable **Superuser Access** to grant superuser privileges to the edb_admin role.
-- If the TDE enabled cluster creation is in progress and the process is waiting providing **Encryption Key Error** next to the cluster name on the clusters page:
-  - Select the cluster name and go to the cluster's home page.
-  - See the **Action required: grant key permissions to activate the cluster** on the cluster's overview tab.
-  - Copy the **Principal** and add it to your key policy to provide **kms:Encrypt** and **kms:Decrypt** permission. 
+- To enable and use TDE for a cluster, the encryption key must be enabled and added at the project level before creating a cluster. 
+  To add a key, see [Adding a TDE key at project level](../../administering_cluster/projects.mdx/#adding-a-tde-key).
+- To enable and use TDE for a cluster, you must complete the configuration on the platform of your key management provider after creating a cluster. See [Completing the TDE configuration](#completing-the-TDE-configuration) for more information.
 !!!
 
+#### Completing the TDE configuration
+
+To complete the TDE key configuration after a TDE-enabled cluster has been created, you will have to complete the configuration by cross-entering the BA key information into your key management platform. The UI reminds you about this step by displaying a **Waiting for access to encryption key** state next to the cluster name on the clusters page. To complete the key configuration:
+
+1.  Select the cluster name to access the cluster's page, and see the **Action required: grant key permissions to activate the cluster**.
+
+1.  Grant permisions for the BigAnimal key in the console of your key management provider: 
+
+    <details><summary>AWS</summary>
+
+    1. Copy the **Principal** identifier to your clipboard. 
+    1. Go to the AWS console, and navigate to the **Key Management Service**
+    1. Select **Customer-managed keys**, and **Edit policy**.
+    1. Paste the **Principal** identifier you copied to your clipboard into the `Principal.AWS` field. 
+    1. In the `Principal.Action` field, provide **kms:Encrypt** and **kms:Decrypt** permissions and **Save**.
+
+    </details>
+
+    <details><summary>GCP</summary>
+
+    1. Copy the **service account** to your clipboard.
+    1. Go to the Google Cloud console, select **Security**, **VIEW BY PRINCIPALS**, **GRANT ACCESS**.
+    1. Paste the service account into the **New principals** field.
+    1. Assign the `Cloud KMS CryptoKey Decrypter` and `Cloud KMS CryptoKey Encrypter` roles and save. 
+
+    </details>
+
+    <details><summary>Azure</summary>
+
+    1. Copy the **MSI Workload Identity** to your clipboard. 
+    1. Got to the Microsoft Azure console, and navigate to **Key vaults**. 
+    1. Select the key, **Access policies**, and **Create**.
+    1. In **Permissions**, select **Encrypt** and **Decrypt**.
+    1. In **Principal**, paste the MSI Workload Identity you copied to your clipboard and finish creating the policy.
+
+    </details>
+    
+
+
+#### Completing the TDE configuration 2
```

Comment:
This is option 2, with simplified instructions per cloud provider on how to finalize the configuration. 
It is less accurate, but we would not have to worry about the instructions becoming outdated without us noticing.